docs/conf/scalar-prometheus-custom-values.yaml

```
@@ -34,6 +39,25 @@ grafana:
       # -- https://grafana.com/docs/grafana/latest/administration/configuration/#allow_embedding
       allow_embedding: false
 
+  # -- Additional data source configurations
+  additionalDataSources:
+  - name: Prometheus
+    type: prometheus
+    uid: prometheus
+    url: http://scalar-monitoring-kube-pro-prometheus.monitoring.svc.cluster.local:9090/
+    access: proxy
+    editable: false
+    isDefault: false
+    jsonData:
+      timeInterval: 30s
+  # - name: Loki
+  #   type: loki
+  #   uid: loki
+  #   url: http://scalar-logging-loki.monitoring.svc.cluster.local:3100/
```

If Loki can be expected to be deployed in the same namespace with Grafana, you should avoid specifying namespace.

```suggestion
  #   url: http://scalar-logging-loki:3100/
```

---

We expect that Loki is deployed in the same namespace with Grafana.
Also, in our document, it describes that we deploy Loki and Grafana in the same namespace.

So, I applied this suggestion. Thank you!